src/utils/transaction.ts

```diff
@@ -143,28 +156,29 @@ export const klimaDaoStaking = async (
 
     return { result: { transactions, provider: 'LiFi' } };
   } catch (e) {
-    return { errorMessage: 'Failed to get staking exchange transaction' }
+    return { errorMessage: 'Failed to get staking exchange transaction' };
   }
-
-}
+};
 
 export const buildCrossChainAction = async (
   sdk: EtherspotSdk,
-  transactionBlock: ITransactionBlock,
+  transactionBlock: ITransactionBlock
 ): Promise<{ errorMessage?: string; crossChainAction?: ICrossChainAction }> => {
   const createTimestamp = +new Date();
   const crossChainActionId = uniqueId(`${createTimestamp}-`);
 
-  if (transactionBlock.type === TRANSACTION_BLOCK_TYPE.KLIMA_STAKE
-    && !!transactionBlock?.values?.fromChainId
-    && !!transactionBlock?.values?.fromAssetAddress
-    && !!transactionBlock?.values?.fromAssetDecimals
-    && !!transactionBlock?.values?.fromAssetSymbol
-    && !!transactionBlock?.values?.amount
-    && !!transactionBlock?.values?.receiverAddress
-    && !!transactionBlock?.values?.routeToKlima
-    && !!transactionBlock?.values?.routeToUSDC
-    && !!transactionBlock?.values?.receiveAmount) {
+  if (
+    transactionBlock.type === TRANSACTION_BLOCK_TYPE.KLIMA_STAKE &&
+    !!transactionBlock?.values?.fromChainId &&
+    !!transactionBlock?.values?.fromAssetAddress &&
+    !!transactionBlock?.values?.fromAssetDecimals &&
+    !!transactionBlock?.values?.fromAssetSymbol &&
+    !!transactionBlock?.values?.amount &&
+    !!transactionBlock?.values?.receiverAddress &&
+    !!transactionBlock?.values?.routeToKlima &&
+    !!transactionBlock?.values?.routeToUSDC &&
+    !!transactionBlock?.values?.receiveAmount
+  ) {
```

Comment:
sorry, these and similar changes remain unnecessary for the scope :(

Reply:
most of codebase is with logical operators by beginning of each line so would be good to keep it consistent

Reply:
@poocart are you using any formatter like Prettier? I think it's worth running the entire codebase through a formatter that we all agree on and then have that be our default formatting option on save, so that we can all get consistent code 

otherwise these sorts of things will keep happening, especially with more and more hands touching the code